services/registry/server.js

```diff
@@ -35,7 +35,7 @@ const main = () => {
 
   // Docker gives containers 10 seconds to handle SIGTERM
   // before sending SIGKILL. Close all current connections
-  // graceully and exit with 0.
+  // gracefully and exit with 0.
```

Comment:
i am so good at words

The same typo exists in storage/server.js and web/server.js, could you correct it there, too?

Reply:
Done.